gluten-ut/spark34/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/GlutenParquetColumnIndexSuite.scala

```diff
@@ -38,7 +38,25 @@ class GlutenParquetColumnIndexSuite extends ParquetColumnIndexSuite with GlutenS
         "cast(id as float) as _5",
         "cast(id as double) as _6",
         "cast(id as decimal(20,0)) as _7",
-        "cast(cast(1618161925000 + id * 1000 * 60 * 60 * 24 as timestamp) as date) as _9"
+        "cast(cast(1618161925 + id * 60 * 60 * 24 as timestamp) as date) as _9"
```

Comment:
Why do we need this change here? Is it because larger timestamp will overflow in Velox? Could add some comments like "// Change 1618161925000 to 1618161925 because ... " for explanations?

Reply:
@marin-ma, it's for avoiding reaching the limitation of Velox. Any timestamp past 2037-11-01 09:00:00 is not supported by Velox.